tensorboard/webapp/feature_flag/store/feature_flag_reducers_test.ts

```diff
@@ -106,4 +107,47 @@ describe('feature_flag_reducers', () => {
       );
     });
   });
+
+  describe('#globalSettingsLoaded', () => {
+    it('sets dark mode overrides when global settings include it', () => {
+      const prevState = buildFeatureFlagState({
+        isFeatureFlagsLoaded: true,
+        flagOverrides: buildFeatureFlag({
+          enableDarkModeOverride: false,
+        }),
+      });
+
+      const state1 = reducers(
+        prevState,
+        globalSettingsLoaded({
+          partialSettings: {},
+        })
+      );
+      expect(state1.flagOverrides!.enableDarkModeOverride).toBe(false);
```

Comment:
why this is `false` but not default value `null`?
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/feature_flag/store/feature_flag_store_config_provider.ts;l=25

Reply:
It is because the initial value, in the test, is set to `false` in L116.
```
        flagOverrides: buildFeatureFlag({
          enableDarkModeOverride: false,
        }),
```